pipeline/alert.go

```diff
@@ -733,6 +737,65 @@ type HipChatHandler struct {
 	Token string
 }
 
+// Send the alert to JIRA.
+//
+// Example:
+//    [jira]
+//      enabled = true
+//      url = ""
+//      username = ""
+//      password = ""
+//      project = ""
+//      issue_type = ""
+//      priority_warn = ""
+//      priority_crit = ""
+//      global = false
+//
+// Example:
+//    stream
+//         |alert()
+//             .jira()
+//
+//
+// If the 'jira' section in the configuration has the option: global = true
+// then all alerts are sent to JIRA without the need to explicitly state it
+// in the TICKscript.
+//
+// Example:
+//    [jira]
+//      enabled = true
+//      url = "https://your.jira.host"
+//      username = "test-username"
+//      password = "test-password"
+//      project = "TEST"
+//      issue_type = "Bug"
+//      priority_warn = "Normal"
```

Comment:
For now I would leave only WARNING/CRITICAL. It's hard for me to imagine reason why JIRA ticket should be created in case of priority OK/INFO.